server/jetstream_errors_generated.go

```diff
@@ -2124,3 +2150,33 @@ func NewJSTemplateNameNotMatchSubjectError(opts ...ErrorOption) *ApiError {
 
 	return ApiErrors[JSTemplateNameNotMatchSubjectErr]
 }
+
+// NewJsConsumerDuplicateFilterSubjectsError creates a new JsConsumerDuplicateFilterSubjects error: "consumer cannot have both FilterSubject and FilterSubjects specified"
+func NewJsConsumerDuplicateFilterSubjectsError(opts ...ErrorOption) *ApiError {
+	eopts := parseOpts(opts)
+	if ae, ok := eopts.err.(*ApiError); ok {
+		return ae
+	}
+
+	return ApiErrors[JsConsumerDuplicateFilterSubjects]
+}
+
+// NewJsConsumerMultipleFiltersNotAllowedError creates a new JsConsumerMultipleFiltersNotAllowed error: "consumer with multiple subject filters cannot use subject based API"
+func NewJsConsumerMultipleFiltersNotAllowedError(opts ...ErrorOption) *ApiError {
+	eopts := parseOpts(opts)
+	if ae, ok := eopts.err.(*ApiError); ok {
+		return ae
+	}
+
+	return ApiErrors[JsConsumerMultipleFiltersNotAllowed]
+}
+
+// NewJsConsumerOverlappingSubjectFiltersError creates a new JsConsumerOverlappingSubjectFilters error: "consumer subject filters cannot overlap"
+func NewJsConsumerOverlappingSubjectFiltersError(opts ...ErrorOption) *ApiError {
```

Comment:
```suggestion
func NewJSConsumerOverlappingSubjectFiltersError(opts ...ErrorOption) *ApiError {
```

Reply:
I think @Jarema will do in separate PR, but great feedback!